python/tvm/dlight/benchmark/utils.py

```diff
@@ -110,6 +119,11 @@ def default_dym_var_sample_func(dym_var_dict: Dict[str, str]) -> Dict[str, int]:
     ----------
     dym_var_dict : Dict[str, str]
         Dynamic shape variable dictionary, e.g., {"n": "int32", "m": "int32"}
```

Comment:
Is there any case where we express shape values other than "int64"?

Reply:
I don't think so, the data types I've seen are only `int32` and `int64`. Given this is part of the input shapes, it's expected to be integer so I think it also makes sense if we just remove the data type annotation and only use a set to store the variables.

Reply:
interesting. do you know where we use `int32`? It would be nice to standardize it if there is no specific reason behind. Then, I believe we can remove this dict and simplify.